templates/jupyter/jupyter-notebook-deployment.template.yml

```diff
@@ -27,6 +27,8 @@ spec:
         - containerPort: 8888
           protocol: TCP
         env:
+        - name: MASTER
+          value: spark://spark-master:7077
```

Comment:
Should this be a config parameter so it can be templated in to all Notebook containers?